app/controllers/api/v0/products_controller.rb

```diff
@@ -130,19 +128,20 @@ def paged_products_for_producers(producer_ids)
           includes(variants: [:product, :default_price, :stock_items]).
           where(supplier_id: producer_ids).
           by_producer.by_name.
-          ransack(params[:q]).result.
-          page(params[:page]).per(params[:per_page])
+          ransack(params[:q]).result
       end
 
       def render_paged_products(products, product_serializer = ::Api::Admin::ProductSerializer)
+        @pagy, products = pagy(products, items: params[:per_page] || DEFAULT_PER_PAGE)
```

Comment:
It's the third time I encounter this `||` clause. I think it deserves some DRY.

Reply:
Should I refactor these into a nice method?